src/app/ssmt/ssmt-report/ssmt-report.component.ts

```diff
@@ -157,11 +159,20 @@ export class SsmtReportComponent implements OnInit {
 
   calculateResultsWithMarginalCosts(ssmt: SSMT, outputData: SSMTOutput, baselineResults?: SSMTOutput): SSMTOutput {
     let marginalCosts: { marginalHPCost: number, marginalMPCost: number, marginalLPCost: number };
+
     if (ssmt.name == 'Baseline') {
       marginalCosts = this.ssmtService.calculateBaselineMarginalCosts(ssmt, outputData, this.settings);
+    } else if (!ssmt.valid.isValid) {
+        outputData = baselineResults;
```

Comment:
This is setting the modification output to the baseline results, we don't want to do that since it's the incorrect results.

Shouldn't have this else if statement..